### PR TITLE
[RW-6920][risk=no] Disable access level dropdown for current user

### PR DESCRIPTION
**Pull request by ericsong**
Before
image

After
image

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/workspace/workspace-share.tsx [hunk 1]
```diff
@@ -12,7 +18,7 @@ import * as fp from 'lodash/fp';
 import * as React from 'react';
 
 import {
-  BillingAccountType,
+  BillingAccountType, Profile,
```
Comment (als364) [inline ui/src/app/pages/workspace/workspace-share.tsx:21]: nit: put new import on next line

File: ui/src/app/pages/workspace/workspace-wrapper/component.ts [hunk 2]
```diff
@@ -169,9 +168,6 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
         this.accessLevel = workspace.accessLevel;
       }
     }));
-    this.subscriptions.push(profileStore.subscribe(({profile}) => {
```
Comment (als364) [inline ui/src/app/pages/workspace/workspace-wrapper/component.ts:172]: Wow, was this dropdown really the only thing that used this?

**Review by als364 — APPROVED** [review]
Thank you for doing this! This seems like exactly the sort of bug that should motivate us to close out the React conversion so it can't happen again :)

File: ui/src/app/pages/workspace/workspace-wrapper/component.ts (hunk 2 above)
Comment (ericsong) [reply ui/src/app/pages/workspace/workspace-wrapper/component.ts:172]: yeah.. turns out a lot of the stuff in here is for one offs like this. will be gone soon!

File: ui/src/app/pages/workspace/workspace-share.tsx (hunk 1 above)
Comment (ericsong) [reply ui/src/app/pages/workspace/workspace-share.tsx:21]: ill drop this change into my next follow up PR which is coming right up

Comment (ericsong) [reply ui/src/app/pages/workspace/workspace-share.tsx:21]: actually, my tests failed so I'll put them in here since I'm making changed anyways

**Comment by ericsong** [comment]
@als364 np! glad to hop on the conversion train. just made some changes to fix the test

**Review by als364 — APPROVED** [review]

**Pull request merged**